src/app/shared/components/terminal/terminal-status-bar/style.scss

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@use 'mixins';
+
+.toolbar-status-container {
+  @include mixins.size(100%, 3rem);
+
+  align-items: center;
+  box-sizing: border-box;
+  display: flex;
+  padding: 0 1.75rem;
+
+  & button[mat-flat-button],
+  & button[mat-icon-button], {
+    @include mixins.size(auto, 1.8rem);
```

Comment:
We don't use relative size units (`em`, `rem` etc.) so any particular reason for using them here?